### ISSUE TITLE
Add a JSoC Alumni Page 

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaLang/www.julialang.org#1225]
It might be cool to have a list by year of all of Season of Contribution students. If any past JSoC/GSoC students want to tackle this, feel free too, otherwise, I will get to it eventually.

**Comment by StarTrooper08** [comment]
Yes , I was too as a beginner struggling and wanted to know actually how does the this GSoC and GSoD works .And best way is to have a look over past year GSoC and GSoD contributors experience and how they did it. 
It would be nice if we have this website surely this will help.

**Comment by logankilpatrick** [comment]
I was thinking maybe like a gallery view of peoples profile icons

**Comment by logankilpatrick** [comment]
I was thinking a grid of cards which use this: https://getbootstrap.com/docs/5.1/components/card/

**Comment by xlxs4** [comment]
The archives were updated in https://github.com/JuliaLang/www.julialang.org/pull/2323
